odoo/addons/base/models/res_config.py

```diff
@@ -555,6 +559,7 @@ def set_values(self):
         """
         self = self.with_context(active_test=False)
         classified = self._get_classified_fields()
+        current_settings = self.default_get(list(self.fields_get()))
```

Comment:
Why do you use `self.fields_get()` (which retrieves translations) instead of `self._fields`?